helmfile/apps/livekit/charts/livekit-server/Chart.yaml

```
@@ -0,0 +1,22 @@
+#
+# SPDX-License-Identifier: APACHE-2.0
+
+annotations:
+  category: video
+  licenses: Apache-2.0
+apiVersion: v2
+appVersion: 0.1.0
+dependencies:
+  - name: common
+    repository: "file://../../../common/charts/common"
+    version: 2.31.1
+description: video conferencing backend platform
+home: https://bitnami.com
+keywords:
+  - Video
+  - livekit
+maintainers:
+  - name: mijn-bureau-infra
+    url: "https://minbzk.github.io/mijn-bureau/"
+name: livekit_server
```

```suggestion
name: livekit-server
```

In general I would change the name to livekit-server to be more consistent. Ofcourse this is all optional.